api/go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.16.0
 	go.opentelemetry.io/otel/trace v1.16.0
 	go.opentelemetry.io/proto/otlp v0.19.0
-	golang.org/x/crypto v0.6.0
+	golang.org/x/crypto v0.9.0
```

Comment:
This is mostly moot since we replace crypto in the base go.mod, but I guess it's alright.